<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_XIANGSHAN_main</class>
 <widget class="QDialog" name="qt_XIANGSHAN_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_dish">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="public_scale_led" name="widget_scale_led" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_dishtotal_title">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>金额</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton_type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dishname_title">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton_type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dishnum_title">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton_type</string>
   </property>
  </widget>
  <widget class="tableview_item" name="tableView_m">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_shop_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>店铺信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton_type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dishprice_title">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>单价</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton_type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_state_title">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton_type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_shop">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_settings">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/hightcontrast/emblem-system.png</normaloff>:/image/hightcontrast/emblem-system.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_shutdown">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/shutdown.png</normaloff>:/image/shutdown.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tableview_item</class>
   <extends>QTableView</extends>
   <header>tableview_item.h</header>
  </customwidget>
  <customwidget>
   <class>public_scale_led</class>
   <extends>QWidget</extends>
   <header>public_scale_led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
